Dockerfile

```diff
@@ -18,6 +22,9 @@ EXPOSE 8080
 
 USER ${USERMAP_UID}
 RUN yarn install --quiet
+RUN yarn add braces@3.0.3
+RUN yarn add semver@7.5.2 
+RUN yarn install --quiet
```

Comment:
Why are we repeating the code that ran in line 24?

Reply:
@hemantgoswami Removed entries from line 25 to 27. Please review PR.